### DISCUSSION TITLE
Deleting a hidden album unhides the photos

### DISCUSSION THREAD

**Discussion by Nohus** [ente-io/ente#1606] [Enhancements]
### Description

After deleting a hidden album, with the "keep photos" option, the photos become unhidden.

### Version

0.8.80

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - Android

**Comment by vishnukvmd** [comment]
Hey @Nohus, this is behaving as intended, because here "hidden" is a property of the album and not the items inside it.

But I do understand that we should either warn the user or hide individual items in this scenario. Thanks for the feedback!

**Reply by Nohus** [reply]
If it's a property of the album, then why removing a photo from a hidden album makes the photo hidden? You can see the inconsistency in behavior. : )

**Reply by vishnukvmd** [reply]
Fixed here: https://github.com/ente-io/ente/pull/1640
